crates/isograph_schema/src/process_client_field_declaration.rs

```diff
@@ -199,6 +199,7 @@ pub struct UserWrittenClientFieldInfo {
 pub enum ClientFieldVariant {
     UserWritten(UserWrittenClientFieldInfo),
     ImperativelyLoadedField(ImperativelyLoadedFieldVariant),
+    Link,
```

Comment:
Thinking out loud... yes, a link field is a client field. The id aspect of it is magical, though, so yes, it sounds like it should be a separate `ClientFieldVariant`. Okay. Sgtm.